engine/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM python:3.9-alpine
-RUN apk add bash python3-dev build-base linux-headers pcre-dev mariadb-connector-c-dev openssl-dev libffi-dev git
-RUN pip install uwsgi
+FROM python:3.9-alpine AS prod
 
 WORKDIR /etc/app
-COPY ./requirements.txt ./
```

Comment:
`COPY ./requirements.txt ./` was redundant as there is a `COPY ./ ./` statement down on line 10

Reply:
This is intentional: https://stackoverflow.com/a/34399661

Reply:
good catch, I'll revert this and add a comment